<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualBolus1</class>
 <widget class="QWidget" name="ManualBolus1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="carbsLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>179</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Carbs (grams)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="carbsValue">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="fetchBloodGlucoseButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Get BG from CGM</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="bloodGlucoseValue">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.250000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="bloodGlucoseLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>169</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Blood Glucose (mmol/L)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="extendedDeliveryPctSlider">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>500</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="extendedDeliveryLabel">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>450</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>How much (in %) to deliver later?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deliveryPctSliderLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Deliver all now</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="applyBolusButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>720</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Manual Bolus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
